Add support for Xen in libvirtd service file

libvirtd should be started after the xencommons service.

libvirtd should also conflict with the xendomains service.
On SUSE distros, we promote libvirt and all the libvirt-based
tools. If a user installs libvirt on their SUSE Xen host, then
libvirt should be king and override xendomains. See bsc#1015348

Index: libvirt-5.6.0/src/remote/libvirtd.service.in
===================================================================
--- libvirt-5.6.0.orig/src/remote/libvirtd.service.in
+++ libvirt-5.6.0/src/remote/libvirtd.service.in
@@ -15,6 +15,8 @@ After=local-fs.target
 After=remote-fs.target
 After=systemd-logind.service
 After=systemd-machined.service
+After=xencommons.service
+Conflicts=xendomains.service
 Documentation=man:libvirtd(8)
 Documentation=https://libvirt.org
 
